packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue

```diff
@@ -264,6 +251,24 @@ export default defineComponent({
       emit('updateLink', { linkShare })
     }
 
+    const showDatePickerModal = () => {
+      const currentDate = DateTime.fromISO(props.linkShare.expirationDateTime)
+
+      dispatchModal({
+        title: $gettext('Set expiration date'),
+        hideActions: true,
+        customComponent: DatePickerModal,
+        customComponentAttrs: () => ({
```

Comment:
You could introduce a `isClearable` prop on the modal and set it to `false` here. 🤔 If I enter the date picker modal by clicking on `Expires in xyz` from the context menu of a link, it's no valid input to have the date cleared. For that I need to click the big `X` button in the context menu.

Reply:
Added the prop to oc-datepicker compontent, but we agreed, that the x is useful